sciencebox/values.yaml

```diff
@@ -587,6 +583,120 @@ swan:
     #       cert:
     #       key:
 
+## cbox web
+cernbox-web:
+  initContainers:
+    - name: install-web
+      image: busybox
+      command: ["/bin/sh", "-c"]
+      args: 
+        - wget -O /var/www/web/web-tar.gz https://github.com/cernbox/web-release/releases/download/v0.0.9/web.tar.gz;
+          tar -xf /var/www/web/web-tar.gz -C /var/www/web/;
+          rm -rf /var/www/web/web.tar.gz;
+          wget -O /var/www/web/theme.json https://new.cernbox.cern.ch/cernbox/theme-cernbox-0.1.7/theme.json;
+          mkdir -p /var/www/web/cernbox/theme-cernbox-0.1.7/assets;
+          wget -O /var/www/web/cernbox/theme-cernbox-0.1.7/assets/cernbox-abstract.svg https://raw.githubusercontent.com/cernbox/branding/master/cernbox-abstract.svg;
+          wget -O /var/www/web/cernbox/theme-cernbox-0.1.7/assets/cernbox-lrg.svg https://raw.githubusercontent.com/cernbox/branding/master/cernbox-lrg.svg;
```

Comment:
You can also add these assets as default (and the `theme.json` web location) in the values so anyone deploying ScienceBox could customize its deployment with their assets.